play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/controllers/Assets.scala

```diff
@@ -27,5 +26,5 @@ import play.api.http.HttpErrorHandler
  * [RuntimeException: java.lang.NoSuchMethodError: controllers.ReverseAssets.versioned(Ljava/lang/String;)Lplay/api/mvc/Call;]
  * when using Assets.
  */
-@Singleton
+@Singleton // TODO migrate off deprecated AssetsBuilder once we drop Play 2.8 support
```

Comment:
just to remind people who see the deprecation warning that we can't do this just yet 🙃